core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -234,6 +235,7 @@ public void seek(long pos) throws IOException {
       return;
     }
     Preconditions.checkArgument(pos >= 0, PreconditionMessage.ERR_SEEK_NEGATIVE, pos);
+    // Why cannot I seek to the end of a file?
```

Comment:
yes. but we can use 10 to show that it is EOF? We allow this in BufferedBlockInStream.java (line 146). The current implementation doesn't allow the client to seek to EOF